src/modules/DressMe/UserSizeSelection.tsx

```diff
@@ -39,3 +45,7 @@ const Select = styled.select`
   border-color: white;
   box-shadow: 0 4px 8px 0 rgb(0 0 0 / 10%), 0 6px 20px 0 rgb(0 0 0 / 1%);
 `;
+
+function setFieldValue(arg0: string, value: any): void {
```

Comment:
what's this function?

you probably used Intellisense (what's helping you with importing stuff that aren't imported yet) and you accidentally chose 'create a function with this name' or something like that